src/pattern-library/components/patterns/data/DataTablePage.tsx

```diff
@@ -488,6 +488,35 @@ export default function DataTablePage() {
             </Library.Demo>
           </Library.Example>
 
+          <Library.Example title="borderless">
```

Comment:
I think ideally we would just link to the `Table.borderless` prop documentation here, to avoid the risk of it getting out of sync. This is awkward right now because individual prop docs in the pattern library don't have their own fragment headers that you can conveniently link to.